### PR TITLE
Display roles in Profile and Users pages

### PR DESCRIPTION
**Pull request by allysonjp715**

### CODE REVIEW

**Comment by allysonjp715** [comment]
[Display roles of users in UI (complete user profile page or simply in the navbar)](https://trello.com/c/4lk6NZpO/147-display-roles-of-users-in-ui-complete-user-profile-page-or-simply-in-the-navbar)

**Comment by allysonjp715** [comment]
[Add page in UI for admins to assign roles to people](https://trello.com/c/zzLbt5u5/146-add-page-in-ui-for-admins-to-assign-roles-to-people)

File: verification/curator-service/ui/src/components/Users.tsx [hunk 1]
```diff
@@ -39,7 +40,9 @@ export default class Users extends React.Component<{}, UsersState> {
         return (
             <div>
                 {this.state.users.map(user =>
-                    <div key={user.googleID}>{user.name}</div>)}
+                    <div key={user.googleID}>
+                        {user.name}: {user.roles?.map(role => `${role}, `)}
```
Comment (attwad) [inline verification/curator-service/ui/src/components/Users.tsx:44]: Use join(', ') instead to avoid the last comma?

Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Users.tsx]: Ah yes of course, done.

**Pull request merged**